src/sass/base/_global.scss

```diff
@@ -6,3 +6,11 @@
   padding: 0;
 }
 
+// had to add this global class to be able to move footer to the bottom
+
+body {
```

Comment:
**body styles**
I assume you did this to push your footer to the bottom along with the absolute positioning but you dont need to add any styles to the body. The footer will adjust as the main content as it is loaded so we can remove these styles.